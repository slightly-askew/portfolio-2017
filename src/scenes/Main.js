import React from "react";
import Header from "../features/header";
import insertGlobalStyles from "../global/style/globalStyles";

insertGlobalStyles();

export const Main = props => <Header {...props} />;
