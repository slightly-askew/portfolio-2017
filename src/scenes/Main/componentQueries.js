//@flow

import componentQueries from "react-component-queries";

export const componentQuery = componentQueries({
  queries: [
    ({ width }) => ({
      queries: {
        device: width < 728 ? "mobile" : "desktop"
      }
    })
  ]
});

//always wrap result in a queries object.
