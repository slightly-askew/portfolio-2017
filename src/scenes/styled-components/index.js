//@flow

export { MainWrapper } from "./MainWrapper";
