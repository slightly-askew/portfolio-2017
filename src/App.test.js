import React from "react";
import ReactDOM from "react-dom";
import { Main } from "./scenes/Main";

import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Main />, div);
});

it("renders a snapshot", () => {
  const tree = renderer.create(<Main />).toJSON();
  expect(tree).toMatchSnapshot();
});
