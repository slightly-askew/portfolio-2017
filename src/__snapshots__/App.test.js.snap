// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="jhXwPk"
>
  <h1
    className="exSSoJ"
  >
    Oliver Askew
  </h1>
</div>
`;
